google-cloud-tools-plugin/testSrc/com/google/gct/idea/appengine/cloud/AppEngineCloudConfigurableTest.java

```diff
@@ -155,7 +156,7 @@ private void initCloudConfigurable() {
   private File createTempFile() throws IOException {
     TemporaryFolder tempFolder = new TemporaryFolder();
     tempFolder.create();
-    File executable = new File(tempFolder.newFolder("bin"), "gcloud");
+    File executable = new File(tempFolder.newFolder("bin"), SystemInfo.isWindows ? "gcloud.cmd" : "gcloud");
```

Comment:
If you want to verify the logic of the cross platform command resolution,
I'd rather we have an explicit test for that.  I don't really buy into the
idea that duplication as a testing failsafe is worth the cost.

On Thu, Mar 31, 2016 at 10:37 AM, Etan Shaul <notifications@github.com>
wrote:

> In
> google-cloud-tools-plugin/testSrc/com/google/gct/idea/appengine/cloud/AppEngineCloudConfigurableTest.java
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/557#discussion_r58064574>
> :
>
> > @@ -155,7 +156,7 @@ private void initCloudConfigurable() {
> >    private File createTempFile() throws IOException {
> >      TemporaryFolder tempFolder = new TemporaryFolder();
> >      tempFolder.create();
> > -    File executable = new File(tempFolder.newFolder("bin"), "gcloud");
> > +    File executable = new File(tempFolder.newFolder("bin"), SystemInfo.isWindows ? "gcloud.cmd" : "gcloud");
>
> Sure I can do that. My thought process was that having the duplication in
> the unit tests asserts that we have those commands correct in the actual
> code. Kind of in the same way that its a good idea to duplicate properties
> from our message bundle in the unit tests rather than referring to them
> directly. What do you guys think?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/557/files/4ed192e3e009802ce94ff923ed0ad34ac78d289c#r58064574>
>